src/ploomber/cli/examples.py

```diff
@@ -301,13 +316,14 @@ def download(self, name, output):
 
 @command_endpoint
 @telemetry.log_call('examples')
-def main(name, force=False, branch=None, output=None):
+def main(name, branch=None, force=False, output=None):
```

Comment:
please revert the  arguments to the original order `main(name, force=False, branch=None, output=None)`

Reply:
Fixed